pnpm-lock.yaml

```diff
@@ -7131,7 +7140,6 @@ packages:
 
   /@types/estree/1.0.0:
     resolution: {integrity: sha512-WulqXMDUTYAXCjZnk6JtIHPigp55cVtDgDrO2gHRwhyJto21+1zbVCtOYB2L1F9w4qCQ0rOGWBnBe0FNTiEJIQ==}
-    dev: true
```

Comment:
Do you know why this is removed? I can't see what we're doing that would make it not a dev dependency.